### ISSUE TITLE
Two-step training (feature importance)

### ISSUE THREAD

**Issue by Innixma** [autogluon/autogluon#3195]
### Discussed in https://github.com/autogluon/autogluon/discussions/3075

<div type='discussions-op-text'>

<sup>Originally posted by **mglowacki100** March 22, 2023</sup>
I've use that is quite common and generic, namely I've training dataset with a lot of features (a few thousands) and  I've no control on them, so I train autogluon model and look at feature importance - typically only top 10-30% really matters. Next I re-train autogluon on selected features. Typically doing this procedure once or twice is enough, but some care is required regarding overfitting. 
I wonder if it is possible to add such functionality to autogluon, that could be activated by parameter like `refit_on_top_features`. 
What do you think about it? </div>

**Comment by Innixma** [comment]
(Reposting my answer to the discussion)

We actually have this feature to some extent, but it isn't well documented / tested:

feature_prune_kwargs in https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.fit.html

You might find it useful. In our past testing, it was able to generally enhance the accuracy of the final model. We will probably plan to refactor it slightly prior to v1.0 so it is more officially supported.

**Comment by eromoe** [comment]
@Innixma
How do I get pruned feature now ? I don't see any way in doc.

My case is select features with several rolling split timeseries dataset.
Since good feature may change along the time, I'd like train severl split , and merge the best 30 features in each model.
